core/src/main/scala/stryker4s/config/Config.scala

```diff
@@ -3,15 +3,22 @@ package stryker4s.config
 import java.nio.file.Path
 
 import better.files._
+import ch.qos.logback.classic.Level
 import com.typesafe.config.ConfigRenderOptions
 import pureconfig.ConfigWriter
 
 case class Config(files: Seq[String] = Seq("**/main/scala/**/*.scala"),
                   baseDir: File = File.currentWorkingDirectory,
-                  testRunner: TestRunner = CommandRunner("sbt", "test")) {
+                  testRunner: TestRunner = CommandRunner("sbt", "test"),
+                  logLevel: Level = Level.DEBUG) {
```

Comment:
Can we change the default log level to INFO? Makes more sense to me, and it's also what Stryker does.

Reply:
Yes, I'll do that.